benchmarks/templates/benchmarks/model.html

```diff
@@ -5,61 +5,64 @@
+                                Warning: This model's build status is shown as having failed; this error might be irrelevant <br>
+                                    to results, but reach out to us to confirm the results are legitimate.
+                                </span>
                             {% endif %}
-                        {% endif %}
-        {%  else %}
-        {% endif %}
+                    {% else %}
+              &nbsp;
+                    {% endif %}
+                </div>
+                    <br>
+                        <span class='information'>Console Log:</span>
+                            {# older models from build IDs 0- 301 do not have a parsed log, so view unparsed: #}
+                            {% if model.submission_id < 302 %}
+                                <a class="log_link" href="http://braintree.mit.edu:8080/job/run_benchmarks/{{ model.submission_id }}/consoleText">View Console Log</a> <br>
+                                <span class='fine_print'>
+                                    This is an older model, so a parsed (interactive) console log will not be available <br>
+                                    until the model is re-run on a new submission. In the meantime, you can still
+                                    view a plaintext version with the link provided. <br>
+                                 </span>
+                            {%  else %}
+                                {% if model.build_status == "successful" or model.build_status == "failure"%}
+                                    <a class="log_link" href="http://braintree.mit.edu:8080/job/run_benchmarks/{{ model.submission_id }}/parsed_console/job/run_benchmarks/{{ model.submission_id }}/parsed_console/log.html">View Console Log</a> <br>
+                                {# older models (0-301) with still running status should not be possible #}
+                                {% elif model.build_status == "running" %}
+                                    <span class='fine_print'>
+                                    <br>
+                                        This model's build status is shown as still running; thus, the console log below
+                                        <br> will not be interactive until the model is done running. In the meantime, you can still
+                                        view a plaintext version <br> with the link provided. <br>
+                                    </span>
+                                    <a class="log_link" href="http://braintree.mit.edu:8080/job/run_benchmarks/{{ model.submission_id }}/consoleText">View Console Log</a>
+                                    <br>
+                                    <br>
+                                {% endif %}
+                            {% endif %}
+            </div>
+            {%  else %}
+            {% endif %}
 
         <span class='information'>Reference Link:</span>
         {% if model.reference_identifier %}
```

Comment:
can we please also move this above the fine-print? I.e. have: 
```
<model name>
<reference link>
```

and then the details?